### ISSUE TITLE
The Popover component should not leverage React portals

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#266]
The Popover component should not leverage React Portals - the popover should basically follow the expand/collapse/accordion ARIA pattern, and as a result, requires that interactive elements occur subsequently in the tab order. The user should be able to:

1. Open a popover (maybe a menu)
2. Tab in to the interactive elements in the popover
3. Shift tab back to the trigger and close the popover

**Comment by jonambas** [comment]
Closing - addressed in hibana
